ecs_task_run/client.py

```diff
@@ -43,6 +43,10 @@ def run_task(self, container_definition, task_family):
             cluster=self.cluster_name,
             taskDefinition=new_definition_arn
         )
+
+        if len(task['failures']) > 0:
```

Comment:
```suggestion
        if len(task.get('failures')) > 0:
```
Acho melhor utilizar um `get` nesse caso dado que a chave poderia não existir...